pom.xml

```diff
@@ -7,12 +7,12 @@
     <parent>
         <artifactId>plugins</artifactId>
         <groupId>org.igniterealtime.openfire</groupId>
-        <version>4.3.0</version>
+        <version>4.5.0-SNAPSHOT</version>
```

Comment:
Let's change this to 4.5.0 (which probably was not released yet when this PR was created).